test/OpenTelemetry.Instrumentation.AWS.Tests/OpenTelemetry.Instrumentation.AWS.Tests.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="AWSSDK.BedrockAgentRuntime" Version="4.0.0" />
     <PackageReference Include="AWSSDK.BedrockRuntime" Version="4.0.0" />
     <PackageReference Include="AWSSDK.S3" Version="4.0.0" />
+    <PackageReference Include="Moq" Version="4.20.72" />
```

Comment:
Moq was removed by #1467.

For the minimal usage just create a hand-rolled mock/stub.

Reply:
I got rid of Moq reference and introduced custom mock for IRequestContext which is actually has most of methods and properties as not implemented as they are not required for such tests. In the future in can be extended if someone needs  more properties.